metricbeat/docs/modules/system.asciidoc

```diff
@@ -17,7 +17,7 @@ The System module has these additional config options:
 regexp expressions to filter the processes that are reported. For more complex filtering, you should use the
 `filters` configuration option.
 +
-The following example config returns metrics for all processes: 
+The following example config returns metrics for all processes:
```

Comment:
Looks like some merge issue. Best is to revert all system related changes to HEAD. I don't assume you changed the system module?